controller/volume_controller.go

```diff
@@ -534,6 +530,14 @@ func (vc *VolumeController) ReconcileEngineReplicaState(v *longhorn.Volume, e *l
 		if err := vc.EvictReplicas(v, e, rs, healthyCount); err != nil {
 			return err
 		}
+
+		// Migrate local replica when Data Locality is on
+		if v.Status.State == types.VolumeStateAttached && !isDataLocalityDisabled(v) && !hasLocalReplicaOnSameNodeAsEngine(e, rs) {
+			if err = vc.replenishReplicas(v, e, rs, e.Spec.NodeID); err != nil {
```

Comment:
Why we're using `err = ` instead of `err :=` here?

Reply:
Thank you! Fixed